### PR TITLE
update

### PR DESCRIPTION
**Pull request by mikhaillav**
# Changes

## API changes

- added player.kill()
- updated "version" cmd so now it show mcpe and protocol version

## Types/JSDoc changes

- added "playerKill" event

## Behavior changes

- Type changes that you did here (ignore if you didn't do any changes related to this category)

## Other changes

- updated russian and english lang for working with new "version" cmd

### CODE REVIEW

File: src/player/PlayerInit.js [hunk 1]
```diff
@@ -68,6 +68,24 @@ module.exports = {
 	 * @param {import('frog-protocol').Server} server
 	 */
 	async initPlayer(player, server) {
+        /**
+         * Kill a player
+         */
+        player.kill = function(cause = HungerCause.UNKNOWN) {
```
Comment (kotinash) [inline src/player/PlayerInit.js:74]: should be DamageCause

**Review by kotinash — CHANGES_REQUESTED** [review]

File: src/player/PlayerInit.js (hunk 1 above)
Comment (mikhaillav) [reply src/player/PlayerInit.js:74]: oh, yes

Comment (mikhaillav) [reply src/player/PlayerInit.js]: I did fix

Comment (kotinash) [reply src/player/PlayerInit.js]: good

**Review by kotinash — APPROVED** [review]

**Pull request merged**